docs/NNNN-direct-clean-path-layout.md

```diff
@@ -0,0 +1,451 @@
+`encodeUTF == true`.  
+If you don't want to use the verbose UTF-Code replacements (`encodeUTF == false`), first check your source to make sure, that there's no chance of 
+different names to be mapped on one target. This happens on whitespace replacement as well as on 
+the removal of leading `-`, `~` and ` ` (blank) or trailing ` ` (blank)
+Software which generates the OCFL 
+structure should raise an error in this case. 
+##### Example names, that would map to the same target
+* `~file` => `file`
+* `-file` => `file`
+* ` file` => `file`
+* `file` => `file`
+* `file ` => `file`
+
+#### Object Identifiers
+When using this extension as a storage layout, you must ensure that none of your 
+object identifiers are prefixes of other identifiers. For example, 
+`https://hdl.handle.net/XXXXX/test` and `https://hdl.handle.net/XXXXX/test/blah`. 
+This is a problem because it would result in the storage paths 
+`https/hdl.handle.net/XXXXX/test` and `https/hdl.handle.net/XXXXX/test/blah`, which is 
+invalid because the first object contains the second.
+
+
+## Parameters
+
+### Summary
+
+* **Name:** `encodeUTF`
+    * **Description:** Decides whether "dangerous" characters will be replaced by a 
+      defined replacement string or it's utf code (e.g. `=u0020` for blank char) 
+    * **Type:** bool
+    * **Default:** false
+* **Name:** `maxFilenameLen`
+   * **Description:** Determines the maximum number of characters within parts of the full pathname separated by `/` (files or folders).   
+     A result with more characters will raise an error.  
+    * **Type:** number
+   * **Constraints:** An integer greater than 0
+   * **Default:** 127
+* **Name:** `maxPathnameLen`
+    * **Description:** Determines the maximum number of result characters.
+      A result with more characters will raise an error.  
+    * **Type:** number
+    * **Constraints:** An integer greater than 0
+    * **Default:** 32000
+* **Name:** `replacementString`
+  * **Description:** String that is used to replace non-whitespace characters 
+    which need replacement.  
+    If `encodeUTF == true` only used for replacement of non-UTF8 characters.
+  * **Type:** string
+  * **Default:** "_"
+* **Name:** `whitespaceReplacementString`
+    * **Description:** String that is used to replace [whitespaces](https://en.wikipedia.org/wiki/Template:Whitespace_(Unicode)).  
+      Only used if `encodeUTF == false`.  
+      **Hint:** if you want to remove (inner) whitespaces, just use the empty string
+    * **Type:** string
+    * **Default:** " " (U+0020)
+* **Name:** `fallbackDigestAlgorithm`
+   * **Description:** Name of the digest algorithm to use for generating fallback filenames. 
+     Restricted to Algorithms supported by OCFL
+   * **Type:** string
+   * **Default:** `md5`
+* **Name:** `fallbackFolder`
+    * **Description:** Name of the folder, the fallback files will appear
+    * **Type:** string
+    * **Default:** `fallback`
+* **Name:** `fallbackSubdirs`
```

Comment:
Suggestion: For consistency with [other extensions](https://github.com/OCFL/extensions/blob/b98dcdbe1ec2e7006cb122712884a5bfc092fb34/docs/0003-hash-and-id-n-tuple-storage-layout.md#summary), it may be helpful to change: `fallbackSubdirs` -> `numberOfFallbackTuples`

Reply:
ok, for consistency, it's fine